### PR TITLE
Add BugMuncher to integrations

### PR DESCRIPTION
**Pull request by mattbearman**

### CODE REVIEW

File: integrations.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 #
 -
+  name: "BugMuncher"
+  description: "Website feedback with screenshots. Create todos in Basecamp directly from your website."
```
Comment (javan) [inline integrations.yml]: We use "to-dos" in Basecamp, not "todos". Mind updating your copy to match?

**Comment by javan** [comment]
I can't imagine our customers would comfortable integrating with a service that says "[Fuck Basecamp](https://www.bugmuncher.com/blog/programming-challenge-day-5/)".

**Comment by mattbearman** [comment]
> I can't imagine our customers would comfortable integrating with a service that says "Fuck Basecamp".

That could well be true. However, the blog post in question accurately sums up my feelings while working with the Basecamp API, and I stand by it. If that means BugMuncher is not eligible for the Basecamp 3 Integrations directory so be it.

Either way, BugMuncher's customers will still have the option to integrate with Basecamp.

**Comment by habibalamin** [comment]
Honestly, this is so dumb. This should go in. I don't give two hoots what someone says about Basecamp, if he's providing a Basecamp-integrated service to me that I need, I'll gladly use that service.

The most reason I might have to care about that is concern over the longevity of support if he doesn't enjoy his job. But that's still silly, because no one goes around trying to figure out how much the person they're doing business with enjoys their job, and people generally assume that if support does end, a timely warning, a migration guide, etc. will be provided. It's a non-issue for most people in most cases.

What *does* make me uncomfortable is a Basecamp employee going on Twitter to get support from his followers over… what exactly? Hurt feelings? It just comes across as unprofessional and a mob mentality. The really slimy thing is that it seemed like you were trying to point to this guy as an example of a bad person for saying “fuck Basecamp”. The irony being, of course, that Basecamp darling DHH says, ‘fuck this, fuck that’ all the time.

But hey, you don't have to listen to me; I'm not a customer, and I probably won't be in the market anytime soon.

**Comment by mattbearman** [comment]
Thanks @habibalamin, nice to see a voice of reason here 😁  For the record I love my job, as well I should, as BugMuncher is a one-man company, if I didn't love working on it, it would have died long ago.

I totally agree that @javan taking to Twitter all butthurt to rally support from other Basecamp developers was unnecessary. I called him out on it on Twitter, and predictably received no response.

This is what I wrote in my following monthly income report blog post, and I stand by it:

> I suppose I should try to see it from his perspective, maybe he built Basecamp’s API, and feels my scathing indictment is a personal attack, but that doesn’t change the fact that Basecamp’s API was by far the worst I’ve worked with. I think the main of the reason I found it so aggravating was that he decided to take to Twitter, to garner support from the _Silicon Valley Circle Jerk_, because heaven forbid you speak ill of an SV darling. He even tagged me in the Tweet, but then didn’t respond to my comments. A rather disappointing display really.
>
> One of my favourite things about being a one-man band, is that BugMuncher doesn’t have a ‘brand identity’, it’s just me, I am BugMuncher and BugMuncher is me. BugMuncher can publicly say ‘Fuck Basecamp’, because that’s how I feel, I don’t need to clear it with PR or board members. It’s a very liberating feeling.
>
> I really think companies shouldn’t take themselves so seriously, especially ones like Basecamp that outwardly present a very friendly face. Hell, if someone where to say Fuck BugMuncher I wouldn’t get all bent out of shape about it, I’d instead try to find out why they felt that way, and see what I could do to improve things.

At the end of the day it's up to the Basecamp guys whether or not to include BugMuncher on the integrations page, but I lost a lot of respect for them due to how they handled this.

**Comment by georgeclaghorn** [comment]
Thanks for your feedback. Nine months later, we’re still not in the habit of providing free advertising for people who publicly insult us.

**Pull request closed without merging**